src/api/controllers/getAuthorsController.js

```diff
@@ -0,0 +1,45 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Article } = models;
+const { User } = models;
+/**
+ *
+ *
+ * @export
+ * @class getAuthors
+ */
+export default class getAuthors {
+  /**
+   *
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getAuthors
+   */
+  static async authors(req, res) {
+    const { offset, limit } = req.query;
+    const paginatedUser = await User.findAndCountAll({
```

Comment:
@Ramadhan0 we want authors only. That means we want to get only users who have articles, not `all` users. An inner join would help here. 

Context on inner joins: https://www.w3schools.com/sql/sql_join_inner.asp

Here's one way to do an inner join in Sequelize: https://stackoverflow.com/questions/20460270/how-to-make-join-queries-using-sequelize-on-node-js

Reply:
I changed that.